src/GitHub/Data/Repos.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE CPP               #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
Maybe don't use the `DuplicateRecordFields` extension.  Are there strong reasons to use it?

Reply:
Not really, I used `DuplicateRecordFields` so I could copy-paste `Repo` fields into `CodeSearchRepo` fields. I'll remove it now.